components/tidb_query_datatype/src/codec/mysql/json/mod.rs

```diff
@@ -74,7 +74,7 @@ mod json_merge;
 mod json_modify;
 mod json_remove;
 mod json_type;
-mod json_unquote;
+pub mod json_unquote;
```

Comment:
Why do we need to make `json_unquote` pub here?

Reply:
So we can use json_unquote::unquote_string in tidb_query_expr.